### PR TITLE
Added support for clock64() by using atomicAdd

### PR DESCRIPTION
**Pull request by Sarbojit2019**

### CODE REVIEW

File: include/hip/spirv_hip_devicelib.hh [hunk 1]
```diff
@@ -1170,6 +1170,13 @@ EXPORT unsigned atomicInc(unsigned *address, unsigned val);
 EXPORT unsigned atomicDec(unsigned *address, unsigned val);
 #endif
 
+// TODO: This is a temporary implementation of clock64(),
+//       in future it will be changed with more reliable implementation.
+__device__ static unsigned long long chip_clk_counter = 0;
```
Comment (pjaaskel) [inline include/hip/spirv_hip_devicelib.hh:1175]: Perhaps add underscores to the name as it's an implementation variable?

**Review by pjaaskel — APPROVED** [review]
Otherwise LGTM.

**Comment by pvelesko** [comment]
Also an implementation of `clock`

**Comment by pvelesko** [comment]
can you please remove the change for the HIP submodule

**Pull request merged**